File: cpp/apps/app2/cls/person.cpp
#include "person.h"

#include <iostream>

using namespace std;

void Person::sayHello()
{
  std::cout << "Hello! My name is " + Person::name + "!\n";
  std::cout << "I'm " + std::to_string(Person::age) + " years old now.\n";
  std::cout << "I'm live in " + Person::country + ".\n";

}

File: nodejs/app4/classexample/iface/person.js
 class Person {
    constructor(name, age, country) {
        this.name = name;
        this.age = age;
        this.country = country;
    }

    sayHello() {
        console.log(`Hello, my name is ${this.name}.`);
        console.log(`I'm ${this.age} years old.`);
        console.log(`I'm live in ${this.country}.`);
    }
}

module.exports = { Person };

